lib/surgex/date_time/date_time.ex

```diff
@@ -11,12 +11,17 @@ if Code.ensure_loaded?(Timex) do
       iex> Surgex.DateTime.date_and_offset_to_datetime(~D{2021-10-07}, 5400)
       {:ok, ~U[2021-10-07 01:30:00Z]}
     """
-    @spec date_and_offset_to_datetime(Date.t(), integer(), String.t()) ::
+    @spec date_and_offset_to_datetime(Date.t(), integer(), String.t(), atom()) ::
             {:ok, DateTime.t()} | {:error, term()}
-    def date_and_offset_to_datetime(date, seconds_since_midnight, timezone \\ "Etc/UTC") do
+    def date_and_offset_to_datetime(
+          date,
+          seconds_since_midnight,
+          timezone \\ "Etc/UTC",
+          ambiguous_hour_pref \\ :after
```

Comment:
This is very specific. Maybe `timezone` and `ambiguous_hour_pref` could both form a proper options keyword list, in case it grows in the future (or I want to specify only the latter, but have a default for the former)?